client/src/components/Sub_ProductDetail/Sub_ImageGallery/ImageSlide.jsx

```diff
@@ -46,6 +47,7 @@ function ImageSlide({
         style={{ visibility: (currentImageIndex < images.length - 1) ? 'visible' : 'hidden' }}
         onClick={() => { setCurrentImageIndex(currentImageIndex + 1); }}
         aria-label="next thumbnail"
+        size={22}
```

Comment:
Did not know about the `size` prop until I saw it here (and the searched it up). Learned something new!